src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -4721,10 +4786,6 @@
     text: |
         az network traffic-manager profile update --name MyTmProfile --resource-group MyResourceGroup --status Enabled
     crafted: true
-  - name: Update a traffic manager profile. (autogenerated)
```

Comment:
should we allow  delete existing one when this is no duiplicate?

Reply:
Yeah. Aladdin want us to do so